src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdater.java

```diff
@@ -104,8 +104,7 @@ protected boolean somethingChangedInBloggerData(Blog blog, BloggerEntry bloggerE
 
     private boolean urlFromRssIsValidAndDifferentThanExistingOne(Blog blog,
```

Comment:
I don't understand :) if something changes in data (values in json) for existing blogger then we should update (not save) blogger entry.